pyccel/codegen/pipeline.py

```diff
@@ -105,8 +105,8 @@ def handle_error(stage):
 
     if (language == "c"):
         libs = libs + ['m']
-        # if accelerator == 'openmp':
-        #     libs = libs + ['omp']
+        if accelerator == 'openmp' and sys.platform == "darwin":
+            libs = libs + ['omp']
```

Comment:
Why only for mac? This should be necessary everywhere

Reply:
Travis is raising the error `cannot find -lomp` for linux but I have tested on my linux machine and the library is definitely required